R/Andromeda.R

```diff
@@ -239,21 +231,12 @@ querySqlToAndromeda <- function(connection,
         integerAsNumeric = integerAsNumeric,
         integer64AsNumeric = integer64AsNumeric
       )
-      columnNames <- RSQLite::dbListFields(andromeda, andromedaTableName)
+      columnNames <- colnames(andromeda[[andromedaTableName]])
       newColumnNames <- toupper(columnNames)
       if (snakeCaseToCamelCase) {
         newColumnNames <- SqlRender::snakeCaseToCamelCase(newColumnNames)
       }
-      idx <- columnNames != newColumnNames
-      if (any(idx)) {
-        sql <- sprintf(
-          "ALTER TABLE %s RENAME COLUMN %s TO %s;",
-          andromedaTableName,
-          columnNames[idx],
-          newColumnNames[idx]
-        )
-        lapply(sql, function(x) RSQLite::dbExecute(andromeda, x))
-      }
+      names(andromeda[[andromedaTableName]]) <- newColumnNames
```

Comment:
This required a new Andromeda method and is in the current develop branch of Andromeda.